template/src/App.vue

```diff
@@ -1,35 +1,48 @@
 <template>
   <div id="app">
     <img class="logo" src="./assets/logo.png">
+    {{#if router}}
+    <router-view></router-view>
```

Comment:
I agree with @chrisvfritz. I think we should keep official vue templates as simple as possible. If we are going to add `router` and `vuex`, I'd definitely do it as a separate template. Although on the other hand this means lot's of copy/pasta boiler plate thats already here (all webpack config and stuff).

I'd use https://github.com/egoist/vuepack if I'd need `full-stack-vue` today :)

I don't know, I'd keep it as simple as possible. If adding `vue-router` or `vuex` is hard to this template, I'd start fixing router or vuex (or their docs) instead.